CHANGELOG.md

```diff
@@ -4,6 +4,82 @@ All notable changes to Router will be documented in this file.
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/3397 https://github.com/apollographql/router/pull/3662
+
+## 🐛 Fixes
+
+### Update Deno to resolve Rust Docs generation failure ([Issue #3305](https://github.com/apollographql/router/issues/3305))
+
+The documentation for our [`apollo-router` Crate on Crates.io](https://crates.io/crates/apollo-router), was failing to build because of a dependency's complication errors [in the docs.rs build environment](https://docs.rs/about/builds). 
+
+We've updated to the latest version of Deno (0.200) which has resolved this constraint and allows us to generate our [docs.rs/apollo-router](https://docs.rs/crate/apollo-router/latest) docs once again.
+
+By [@o0Ignition0o](https://github.com/o0Ignition0o) in https://github.com/apollographql/router/pull/3626
+
+### Fix config metrics path and test for subscription callbacks ([Issue #3687](https://github.com/apollographql/router/issues/3687))
+
+Detection of subscription callbacks has been fixed for internal Apollo metrics. This has no user facing impact.
+
+By [@BrynCooke](https://github.com/BrynCooke) in https://github.com/apollographql/router/pull/3688
+
+### GraphQL response processing is now captured under the execution span ([PR #3732](https://github.com/apollographql/router/pull/3732))
+
+This ensures that the response processing is captured under the "execution" span.  Previously, any event in processing would be reported under the supergraph span or — even more arbitrarily — any plugin's span (e.g., Rhai) which was happening in between.
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/3732
+
+## 🛠 Maintenance
+
+### Apollo Uplink connections re-use the existing HTTP client ([Issue #3333](https://github.com/apollographql/router/issues/3333))
+
+A single HTTP client will now be shared between requests of the same type when making requests to [Apollo Uplink](https://www.apollographql.com/docs/federation/managed-federation/uplink/) to fetch supergraphs, licenses and configuration from Studio.  Previously, such requests created a new HTTP client on each periodic fetch which occasionally resulted in CPU spikes, especially on macOS.
+
+By [@BrynCooke](https://github.com/BrynCooke) in https://github.com/apollographql/router/pull/3703
+
+### Add a metric tracking authorization usage ([PR #3660](https://github.com/apollographql/router/pull/3660))
+
+The new metric is a counter called `apollo.router.operations.authorization` and contains the following boolean attributes:
+- `filtered`: the query has one or more filtered fields 
+- `requires_scopes`: the query uses fields or types tagged with the `@requiresScopes` directive
+- `authenticated`: the query uses fields or types tagged with the `@authenticated` directive
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/3660
+
```

Comment:
I think this reflects an additive change during the feature development of authorization, but in terms of this release, authorization is net-new, so it makes less sense to call this "maintenance" out separately here.

I would suggest we could just remove it.

```suggestion
```